2-mandatory/4-space-colonies.js

```diff
@@ -14,8 +14,10 @@
         Logic: Only strings that start with A, and finish with family
   
 */
-
-function getSettlers() {}
+// filter through array to return strings with index that begins with A
+function getSettlers(alphaPlanet) {
+  return alphaPlanet.filter(familyName => familyName.startsWith("A") && familyName.endsWith("family"));
```

Comment:
Hi, your code is very nice, I didn't know about `.startsWith` or `.endsWith` and made my code too long and complicated, thanks for the tip